### DISCUSSION TITLE
Snaps as deps

### DISCUSSION THREAD

**Discussion by soumyaDghosh** [pacstall/pacstall-programs#2274] [Ideas]
In many cases, we need latest version, which maybe is available via snap. Is that possible to make the snap a dep?

**Comment by Elsie19** [comment]
No, Pacstall uses apt as it's backend for packages, not snap. If you want the latest version, try making a pacscript for it.

**Reply by soumyaDghosh** [reply]
Can you make a pacscript for Rust? That seems tough

**Reply by Elsie19** [reply]
You want, like, the rust programming language in a Pacscript?

**Reply by soumyaDghosh** [reply]
I want to install rust via pacstall

**Reply by Elsie19** [reply]
That would kill our github action free time, hmmmm.

**Reply by soumyaDghosh** [reply]
There are a lot of packages with rust as a build_dep, what should I do? Check my another discussion

**Reply by Elsie19** [reply]
Most people don't want to install rust from source as a dependency. That's what Gentoo is for ;)

**Reply by Elsie19** [reply]
Maybe a `rust-bin` package.

**Reply by soumyaDghosh** [reply]
Well, even using rust, the building time is very high, and with less capable machines, that's too heavy also. I think the only thing that we can do is, we need to share these apps prebuilt. We may also share these as debs
